axelor-quality/src/main/java/com/axelor/apps/quality/db/repo/ControlEntryPlanLineManagementRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.axelor.apps.quality.db.repo;
+
+import com.axelor.apps.base.service.exception.TraceBackService;
+import com.axelor.apps.quality.db.ControlEntryPlanLine;
+import com.axelor.apps.quality.service.ControlEntryPlanLineService;
+import com.axelor.apps.quality.service.ControlEntrySampleService;
+import com.axelor.inject.Beans;
+
+public class ControlEntryPlanLineManagementRepository extends ControlEntryPlanLineRepository {
+
+  @Override
+  public ControlEntryPlanLine save(ControlEntryPlanLine entity) {
+
+    try {
+      if (entity.getTypeSelect().equals(TYPE_ENTRY_SAMPLE_LINE)) {
```

Comment:
to avoid a possible NPE:
```suggestion
      if (TYPE_ENTRY_SAMPLE_LINE.equals(entity.getTypeSelect())) {
```